packages/next/src/build/index.ts

```diff
@@ -4277,6 +4277,19 @@ export default async function build(
         .traceAsyncFn(() => telemetry.flush())
 
       await shutdownPromise
+
+      if (NextBuildContext.analyze) {
+        await cp(
+          path.join(__dirname, '../bundle-analyzer'),
+          path.join(dir, '.next/diagnostics/analyze'),
+          { recursive: true }
+        )
+        // Copy routes-manifest.json to the public directory so it can be served
+        await cp(
+          path.join(dir, '.next/routes-manifest.json'),
+          path.join(dir, '.next/diagnostics/analyze/data/routes-manifest.json')
+        )
```

Comment:
This makes it depend on the build output files. Maybe we could write a routes.json ourselves while writing the analyze data